src/main/java/com/openlattice/datastore/directory/controllers/PrincipalDirectoryController.java

```diff
@@ -20,24 +20,12 @@
 
 package com.openlattice.datastore.directory.controllers;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import com.auth0.client.auth.AuthAPI;
 import com.auth0.client.mgmt.ManagementAPI;
 import com.auth0.exception.Auth0Exception;
 import com.auth0.json.mgmt.users.User;
 import com.codahale.metrics.annotation.Timed;
-import com.google.common.eventbus.EventBus;
 import com.openlattice.assembler.PostgresRoles;
-import com.openlattice.authorization.AclKey;
-import com.openlattice.authorization.AuthorizationManager;
-import com.openlattice.authorization.AuthorizingComponent;
-import com.openlattice.authorization.DbCredentialService;
-import com.openlattice.authorization.Permission;
-import com.openlattice.authorization.Principal;
-import com.openlattice.authorization.PrincipalType;
-import com.openlattice.authorization.Principals;
-import com.openlattice.authorization.SecurablePrincipal;
+import com.openlattice.authorization.*;
```

Comment:
Why do you expand `import package.thing.*` in some places, but collapse to it in others? For example, in this file, you go from:

```
import com.openlattice.authorization.AuthorizationManager;	
import com.openlattice.authorization.AuthorizingComponent;	
import com.openlattice.authorization.DbCredentialService;	
import com.openlattice.authorization.Permission;	
import com.openlattice.authorization.Principal;	
import com.openlattice.authorization.PrincipalType;	
import com.openlattice.authorization.Principals;	
import com.openlattice.authorization.SecurablePrincipal;
```
to:
```
import com.openlattice.authorization.*;
```

In another file, you go from:
```
import java.util.*;
```
to
```
import java.util.Collection;
import java.util.LinkedHashSet;
import java.util.List;
import java.util.Map;
import java.util.Set;
import java.util.UUID;
```

Reply:
my intellij import settings are 7 non-static / 10 static